source/vscode/src/projectSystem.ts

```diff
@@ -325,21 +325,3 @@ export async function fetchGithubRaw(
 
   return text;
 }
-
-/**
- * Opens the qsharp.json manifest in vscode for the given document URI.
- *
- * @param documentUri The URI of the document for which to update the manifest profile.
- * @throws Error if the manifest cannot be found or parsed.
- */
-export async function openManifestFile(documentUri: vscode.Uri): Promise<void> {
```

Comment:
Removing this, because I noticed that the caller should already have the `qsharp.json` URI available that they can use directly. We don't have to discover the manifest again. I updated the caller to open the manifest directly.